mmcv/runner/hooks/logger/clearml.py

```diff
@@ -0,0 +1,62 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+
+from ...dist_utils import master_only
+from ..hook import HOOKS
+from .base import LoggerHook
+
+
+@HOOKS.register_module()
+class ClearMLLoggerHook(LoggerHook):
+    """Class to log metrics with clearml.
+
+    It requires `clearml`_ to be installed.
+
+
+    Args:
+        task_init_kwargs (dict): A dict contains the `clearml.Task.init`
```

Comment:
Can we rename `task_init_kwargs` to `init_kwargs` which will keep consistent with other logger hooks?

Reply:
done